canonical-web-and-design/jaas-dashboard
# Remove the Prettier overrides and auto-update code

## Done

Remove the Prettier overrides and auto-update code.
This added trailing commas to all arrays and objects as well as wrapping the arguments in arrow functions with parens.

This is their default and it's how we do it in the other projects so just keeping it consistent.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- It should just work as usual
